expo/gbe/views/create_volunteer_view.py

```diff
@@ -54,6 +95,10 @@ def CreateVolunteerView(request):
                 for window in form.cleaned_data['unavailable_windows']:
                     volunteer.unavailable_windows.add(window)
 
+                for interest_form in formset:
+                    vol_interest = interest_form.save(commit=False)
```

Comment:
They are checked for validity on lines 75 through 85 and the only way to get to this line is line 86 - an if statement checking that validity.